app/service/rest.py

```diff
@@ -338,8 +338,6 @@ def _warn_service_users_about_sms_limit_changed(service_id, data):
     send_notification_to_service_users(
         service_id=service_id,
         template_id=current_app.config["DAILY_SMS_LIMIT_UPDATED_TEMPLATE_ID"]
-        if current_app.config["FF_SPIKE_SMS_DAILY_LIMIT"]
-        else current_app.config["DAILY_LIMIT_UPDATED_TEMPLATE_ID"],
```

Comment:
we'll use the FF to decide when we call `_warn_service_users_about_sms_limit_changed()`